### ISSUE TITLE
Principles instead of requirements

### ISSUE THREAD

**Issue by shsdev** [DILCISBoard/E-ARK-CSIP#118]
To distinguish high level principles from "implementable" requirements, the "general requirements" and all subordinate requirements could be named "principles". It would not be necessary to apply the MOSCOW scheme either.

https://github.com/DILCISBoard/E-ARK-CSIP/blob/master/specification/index.md#31-general-requirements

**Comment by karinbredenberg** [comment]
I like the suggestion.

**Comment by jmaferreira** [comment]
The way I see the common specification is as an object (in OO languages), meaning that the SIP, AIP and DIP inherit the properties of this specification.

This means that the requirements specified at the CS level do not need to be rewritten at the lower level specifications, making them much smaller documents.

Additionally, the behaviour of the CS should not be overwritten by the lower packages (as a principle). For example, if an attribute is mandatory at the CS level e should be mandatory at the SIP, AIP and DIP levels.

In summary, if we follow this understanding, we need the MOSCOW rules in the CS.

**Comment by shsdev** [comment]
I agree with that, i.e. there can be requirements in the CS which are inherited by SIP, AIP and DIP. However, the CS contains "non-implementable", "high-level requirements". So the purpose of this change would be to distinguish them more clearly as "principles" from "implementable/testable" requirements.

**Comment by karinbredenberg** [comment]
As @jmaferreira writes the requirements from the CS IP is inherited of the E-ARK Profiles for SIP/AIP/DIP as well as others implementing the CS IP and creating their own profile. But demands not implemented becomes a principle that we enforce but its not testable as @shsdev writes.

**Comment by kuldaraas** [comment]
I understand it as being more a semantic issue (i.e. what terminology we use for the generic requirements). I agree with @jmaferreira that everything that is written in the CS IP remains valid and relevant for SIP/AIP/DIP, including both high- and low-level requirements.

But at the same time using terminology to differentiate between implementable and generic requirements is a really good thought as well, I also like the term "principles". As such, I could: 
* change "requirements" -> "principles" in Ch3; 
* check Ch3 that there is an explicit statement about the relevance of the principles in not only the CS IP but also the other IPs?

@shsdev @karinbredenberg @jmaferreira -- is that ok?

**Comment by shsdev** [comment]
Sounds good. However, I think that the MOSCOW criteria are not well applicable to the principles (currently "high-level requirements"). So I would suggest to remove them as well.

**Comment by kuldaraas** [comment]
For now I've just renamed requirements to principles in 4.1. But I think the MoSCow criteria is still applicable even if we call them principles -- so let's keep the issue open. 

Btw, @carlwilson I also changed the title of 4.1, hope you can check everything is fine with the ToC's!
